src/rgw/rgw_sync_module_es.cc

```diff
@@ -248,13 +248,13 @@ struct es_index_config {
 };
 
 static bool is_sys_attr(const std::string& attr_name){
-  static constexpr std::initializer_list rgw_sys_attrs = {RGW_ATTR_PG_VER,
-                                                          RGW_ATTR_SOURCE_ZONE,
-                                                          RGW_ATTR_ID_TAG,
-                                                          RGW_ATTR_TEMPURL_KEY1,
-                                                          RGW_ATTR_TEMPURL_KEY2,
-                                                          RGW_ATTR_UNIX1,
-                                                          RGW_ATTR_UNIX_KEY1
+  static constexpr std::initializer_list<const char*> rgw_sys_attrs = {RGW_ATTR_PG_VER,
```

Comment:
again, i'd suggest minimize the code churn to fix the FTBFS with Clang. so instead reformatting the following lines. i think we can just fix the first line.

also, please wrap at 80 chars.